osmoutils/ibc.go

```diff
@@ -11,15 +12,19 @@ import (
 // NewEmitErrorAcknowledgement creates a new error acknowledgement after having emitted an event with the
 // details of the error.
 func NewEmitErrorAcknowledgement(ctx sdk.Context, err error, errorContexts ...string) channeltypes.Acknowledgement {
+	errorType := "ibc-acknowledgement-error"
```

Comment:
nit: This could be a package `const ibcAcknowledgementError` (to avoid reallocating the variable on each function call)